scripts/create-instance.sh

```diff
@@ -106,12 +109,18 @@ cdvirtualenv
 set -o errexit
 set -o nounset
 
-if [[ "$@" != *"--devel"* ]]; then
+# ideally, when we'll have refactored the setup.py:
+#if [ "$FLASK_DEBUG" != 1 ]; then
+#    pip install invenio[ils]
+#else
+#    pip install invenio[ils,extra]
+#fi
+# For the moment, we use an extra requirements file:
 # sphinxdoc-install-invenio-full-begin
 pip install invenio-app-ils[postgresql,elasticsearch2]
 # sphinxdoc-install-invenio-full-end
-else
-    pip install -r "$scriptpathname/../requirements-devel.txt"
+if [ "$FLASK_DEBUG" == 1 ]; then
+    pip install -r "$scriptpathname/../requirements-extra.txt"
```

Comment:
this doesn't work because ENV variable in docker-compose.yml are set for run time, and this script is called at build time...
if you have an idea to make it work for both dev and non-dev...